### DISCUSSION TITLE
casaos install help

### DISCUSSION THREAD

**Discussion by fionnafire** [tenseiken/docker-qbittorrent-wireguard#11] [Q&A]
Can anyone help me understand how to set this up in casaos please? Or link to a guide?

**Comment by tenseiken** [comment]
I'm not familiar with casaos but just reading a little about it, it seems kind of like openmediavault but with more cloud interconnection. It kind of sounds like it uses a "store" interface with limited options by default but there's probably a way to add more docker images. Does it allow you to get to a terminal? If so, you could do it with docker run or docker-compose. If it has a built in container manager (ex: Portainer) you should be able to do it from there too.
